src/_data/how_to/get_connected.yml

```diff
@@ -23,6 +23,37 @@ toc:
           - text: Cal-ITP will provide the materials and support you need to implement data plans on your fleet. To help us better assist you, we ask that you complete our contact form for Cal-ITP support. With this information, we can determine how best to support you with finding the right data plan for your needs, getting buy-in from internal stakeholders, and engaging with vendors.
```

Comment:
Hi @esquared415 - Do you mind sending edit requests as GitHub comments? You can view the content here: https://deploy-preview-465--cal-itp-mobility-marketplace.netlify.app/how-to/get-connected#what-is-firstnet

And then for any sentences that need change, make a comment on the line below by clicking the blue plus sign like this: 

<img width="1160" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/3673236/34ebc098-9d14-43ec-9774-f5ed05465724">

It's easier for me to track changes in GitHub rather than the Google Doc. Thank you!

Reply:
@machikoyasuda @thekaveman @mrose914 Sat down to review all copy, but can't figure out how to navigate to the codeview or whatever view it was that Machiko shows above that allows for tracking comments by clicking blue + sign. Could you point me to that view? (Thank you!)

Reply:
@esquared415 click on the `Files changed` tab at the top of this page (it is the last of 4 tabs, we are currently in the `Conversation` tab).

<img width="770" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/1783439/d748ddc0-8049-4c3b-82b0-64040992fb99">

From there, you get a view of every file, every line that changed. Each line gives you the hover blue icon where you can add a comment.

<img width="867" alt="image" src="https://github.com/cal-itp/mobility-marketplace/assets/1783439/658e908a-c7c9-48ed-8b95-a25c6356371b">

Reply:
hi @esquared415 Here's the direct link to the `Files changed` tab view of this pull request: https://github.com/cal-itp/mobility-marketplace/pull/465/files

Reply:
hi @esquared415 - do you have any edits for this PR and this doc (which is linked in the new changes)? https://docs.google.com/document/d/1f9mvRPx1Zd2lDJgkIiDQjtVF3uxidFYTOHmRAdxLX88/edit https://deploy-preview-465--cal-itp-mobility-marketplace.netlify.app/how-to/get-connected#what-is-firstnet

If you can't get the changes into GitHub, you can send it via Google Doc or anything that's easier for you.

cc @mrose914